### PR TITLE
Accept kw args in Dict.update. Fix #64

### LINKED ISSUE #64: Dict.update does not accept keyword args
Python `dict.update` accepts keywords to update specific keys:

https://docs.python.org/3/library/stdtypes.html#dict.update

Addict `Dict.update` raises an error instead:

```
In [1]: from addict import Dict

In [2]: d = Dict()

In [3]: d.update(red=1)
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-3-5b5e6d65d506> in <module>()
----> 1 d.update(red=1)

TypeError: update() got an unexpected keyword argument 'red'

In [4]: d = {}

In [5]: d.update(red=1)

```

*(last command succedes)*

This should be fixed to have drop-in compatibility with python's `dict`.

### PR DESCRIPTION
**Pull request by tritemio**
Addresses #64

### CODE REVIEW

**Comment by mewwts** [comment]
Hey @tritemio! Thanks a bunch. Would you mind writing a test or two? 😃

File: test_addict.py [hunk 1]
```diff
@@ -357,6 +357,12 @@ def test_update_with_lists(self):
         self.assertDictEqual(org, correct)
         self.assertIsInstance(org.b[0], dict)
 
+    def test_update_with_kws(self):
+        org = Dict(one=1, two=2)
+        someother = Dict(one=3)
+        someother.update(one=1, two=3)
```
Comment (mewwts) [inline test_addict.py]: Should this be `someother.update(one=1, two=2)`?

Comment (tritemio) [inline test_addict.py]: Right!

Comment (tritemio) [inline test_addict.py]: I changed the test to make it fail and I committed the failing version ;). Now fixed.

**Comment by mewwts** [comment]
Thanks @tritemio 👍

**Pull request merged**